proto-google-cloud-talent-v4beta1/clirr-ignored-differences.xml

```diff
@@ -16,4 +16,13 @@
     <className>com/google/cloud/talent/v4beta1/*OrBuilder</className>
     <method>boolean has*(*)</method>
   </difference>
+  <difference>
```

Comment:
Can we add the note to remove this after 0.36.0?

Reply:
done